maat-court-data-api/src/main/java/gov/uk/courtdata/applicant/mapper/RepOrderApplicantLinksMapper.java

```diff
@@ -16,5 +16,5 @@ public interface RepOrderApplicantLinksMapper {
     List<RepOrderApplicantLinksDTO> mapEntityToDTO(List<RepOrderApplicantLinksEntity> repOrderApplicantLinksEntities);
     RepOrderApplicantLinksDTO mapEntityToDTO(RepOrderApplicantLinksEntity repOrderApplicantLinksEntity);
     void updateRepOrderApplicantLinksDTOToRepOrderApplicantLinksEntity(RepOrderApplicantLinksDTO repOrderApplicantLinksDTO, @MappingTarget RepOrderApplicantLinksEntity repOrderApplicantLinksEntity);
-
+    RepOrderApplicantLinksEntity map(RepOrderApplicantLinksDTO repOrderApplicantLinksDTO);
```

Comment:
Can this be a slightly more informative name of what it's mapping from/to?

Reply:
Renamed to mapDTOToEntity to be consistent with the other mapper method mapEntityToDTO